assets/locales/en.json

```diff
@@ -22,10 +22,12 @@
             "link_text":"See all featured projects"
         },
         "dao_proposals":{
-            "title":"DAO ",
+            "title":"Open DAO ",
```

Comment:
question (blocking): Adding spaces to the end of a string is always a little tricky as it can lead to bugs that are difficult to detect. Especially if there is no consistency in place. Can't we add a space when we use the actual entry?  e.g. `{{ title }} {{titleHighlight}}` ?